addons/hr/models/res_users.py

```diff
@@ -17,6 +17,7 @@
     'last_activity_time',
     'can_edit',
     'is_system',
+    'employee_resource_calendar_id',
```

Comment:
@tde-banana-odoo Hi, this is what I was referring to (HR_READABLE_FIELDS). Is this ok? Should I not include the field in one of the lists? I am not certain. Cheers :)